experimental/generation/generator/packages/library/templates/standard/en-us/form-Prompts.en-us.lg

```diff
@@ -0,0 +1,140 @@
+[form.en-us.lg](form.en-us.lg)
+
+>> OnChooseProperty Prompt
```

Comment:
Order the prompts in the same order they are found in the left rail of the current UI.